### PR TITLE
chore(release): publish v0.7.0

### PR DESCRIPTION
**Pull request by petermetz**
Signed-off-by: Peter Somogyvari <peter.somogyvari@accenture.com>

### CODE REVIEW

**Review by jonathan-m-hamilton — APPROVED** [review]
LGTM! 🚀

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,30 @@
...
+* **connector-fabric:** endorsing peers request arg [#1122](https://github.com/petermetz/cactus/issues/1122) ([c6057a7](https://github.com/petermetz/cactus/commit/c6057a7ac508f3bd8da8c9611414a627ff772024)), closes [#1123](https://github.com/petermetz/cactus/issues/1123) [#1130](https://github.com/petermetz/cactus/issues/1130)
+* **connector-fabric:** identity json signing credentials [#1130](https://github.com/petermetz/cactus/issues/1130) ([bc262a2](https://github.com/petermetz/cactus/commit/bc262a24ae654899e8941e7910953c6c561ea778)), closes [#1124](https://github.com/petermetz/cactus/issues/1124)
+* **example:** add README on examples/discounted-cartrade ([2eac8bf](https://github.com/petermetz/cactus/commit/2eac8bf8c819911ded6cbdf2368398fbd6977a64))
+* **examples/discounted_cartrade:** add preferredcustomer-jugdement ([43be168](https://github.com/petermetz/cactus/commit/43be16828581ebf0fabb4b0e8356e052bbcd8760))
+* **google-sm:** added keychain plugin for google secret manager ([1419b2c](https://github.com/petermetz/cactus/commit/1419b2c8c1b12a08e940ba4f0ff824420d88c233)), closes [#983](https://github.com/petermetz/cactus/issues/983)
+
```
Comment (takeutak) [inline CHANGELOG.md:25]: Would you add the following sentence? The PR related with Issue 1181 includes the codes about Validator Indy.
* **validator-indy:** enhance the request feature on Validator-Indy [#1181](https://github.com/hyperledger/cactus/issues/1181

**Comment by takeutak** [comment]
Thanks for contributing! Would you mind if you reflect the above comments on CHANGELOG.md? If you do so, I will approve this PR.

**Comment by petermetz** [comment]
> Thanks for contributing! Would you mind if you reflect the above comments on CHANGELOG.md? If you do so, I will approve this PR.

@takeutak You are welcome! I've added the change requested. 

If you can, in the future please provide commit messages that you want to see in the change log so that I don't have to go manually edit the change logs despite going to great lengths/investing a lot of my time and energy to make the release automation work.

To provide an example: Cases like this is one of the reasons why I always ask people to not have commit messages like "updated readme" - because when that gets put in the change log, it doesn't provide much information about the exact change and is also ambiguous relative to the other "updated readme" commits (if we let this fester).

Also a good reason why it's best to keep pull requests into single commits whenever possible because that usually forces the person to think about what is actually being done on a high level and that helps with coming up with the commit messages.

**Review by takeutak — APPROVED** [review]

**Pull request merged**